### PR TITLE
removed error check while verifying the replication

### PR DESCRIPTION
**Pull request by arindamnayak**
Signed-off-by: Arindam Nayak <arindam.nayak@outlook.com>

### CODE REVIEW

**Comment by deepthi** [comment]
There seems to be a problem with github actions not updating the status correctly on this PR. All tests seem to have passed: https://github.com/vitessio/vitess/actions?query=branch%3Afix_pitr_flaky_test

File: go/test/endtoend/cluster/cluster_util.go [hunk 1]
```diff
@@ -74,11 +74,13 @@ func GetMasterPosition(t *testing.T, vttablet Vttablet, hostname string) (string
...
+// this is used to check replication caught up the changes from master
 func VerifyRowsInTabletForTable(t *testing.T, vttablet *Vttablet, ksName string, expectedRows int, tableName string) {
 	timeout := time.Now().Add(10 * time.Second)
 	for time.Now().Before(timeout) {
-		qr, err := vttablet.VttabletProcess.QueryTablet("select * from "+tableName, ksName, true)
-		require.Nil(t, err)
+		// ignoring the error check, if the newly created table is not replicated, then there might be error and we should ignore it
+		// but eventually it will catch up and if not caught up in required time, testcase will fail
+		qr, _ := vttablet.VttabletProcess.QueryTablet("select * from "+tableName, ksName, true)
 		if len(qr.Rows) == expectedRows {
```
Comment (deepthi) [inline go/test/endtoend/cluster/cluster_util.go:84]: should check for qr nil before accessing Rows. If there is an error, qr will be nil.

Comment (arindamnayak) [reply go/test/endtoend/cluster/cluster_util.go]: Added nil check.

**Comment by deepthi** [comment]
Only unit test failed here: https://github.com/vitessio/vitess/actions?query=branch%3Afix_pitr_flaky_test
with a transient error. Re-running.

**Comment by deepthi** [comment]
Looks like @sougou ran into similar problems and fixed it differently in #6461 
```
		if sqlErr, ok := err.(*mysql.SQLError); ok && sqlErr.Num == mysql.ERNoSuchTable {
			time.Sleep(300 * time.Millisecond)
			continue
		}
		require.NoError(t, err)
```

**Pull request merged**